src/server/index.jsx

```diff
@@ -209,7 +209,7 @@ server
   )
   .get('/*', cspInjectFun, async ({ url, headers, path: urlPath }, res) => {
     try {
-      const { service, isAmp, route, variant } = getRouteProps(routes, url);
+      const { service, isAmp, route, variant } = getRouteProps(routes, urlPath);
```

Comment:
I think we need to carry out some careful checks inside of `getRouteProps` as it seems to do a lot and call a few other functions with regexy stuff. I'll take a look into this later today and let you know what I find out 🙂

Reply:
what is the difference between `url` and `path` here?

Reply:
`req.url` includes querystring - `req.path` excludes it :)